### ISSUE TITLE
CIFAR100 evaluation?

### ISSUE THREAD

**Issue by ashok-arjun** [hongjoon0805/SS-IL-Official#3]
Hi,

Thanks a lot for releasing the code! 

I'd like to know how to tune the hyperparams if I want to evaluate on CIFAR100.

Is there a particular grid or values that I should use to get the best numbers?

Thank you!

**Comment by hongjoon0805** [comment]
HI,

Thanks for giving interest to our project!

In case of CIFAR100, the hyperparameters of SS-IL is as follows:

epochs: 200
lr: 0.1 with lr decay at [80, 120, 160] epochs
weight decay: 0.001

We stress that these hyperparams are optimal only for our machine. If you run our code on different machine, the optimal parameters may be different from ours.

Thanks,
Hongjoon

**Comment by ashok-arjun** [comment]
Thank you very much for the quick reply! I will experiment with these hyperparameters.

**Comment by ashok-arjun** [comment]
@hongjoon0805 Just confirming - the same `resnet18` is used for CIFAR100, right?

**Comment by hongjoon0805** [comment]
We used ResNet18, but the first layer is modified. Please refer the original paper of ResNet.

I guess we replaced 7x7 conv layer with 3x3 conv layer.

**Comment by ashok-arjun** [comment]
Got it, @hongjoon0805. Thank you very much for the quick replies.

**Comment by qsunyuan** [comment]
what about ur CIFAR100 results? Could you pls share the results and settings of the experiment.
